### PR TITLE
BUGFIX: Fix display of not-equal-to on rule graph

### PR DESCRIPTION
**Pull request by 121watts**
- [x] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #1131 

### The problem
Incorrect highlighting of the `not-equal-to` operator.  

### The Solution
Visualize the inverse of the `equal-to` operator.

kapture 2017-10-09 at 13 54 57

### CODE REVIEW

**Comment by 121watts** [comment]
@lukevmorris is this what you had in mind?

**Comment by alexpaxton** [comment]
Naiisssss

File: ui/src/kapacitor/components/RuleGraph.js [hunk 1]
```diff
@@ -53,6 +53,21 @@ export const RuleGraph = React.createClass({
     )
   },
 
+  _getFillColor(operator) {
+    const backgroundColor = 'rgba(41, 41, 51, 1)'
+    const highlightColor = 'rgba(78, 216, 160, 0.3)'
+
+    if (operator === 'outside range') {
```
Comment (goller) [inline ui/src/kapacitor/components/RuleGraph.js]: i think this and the `not equal to` below should be constants.  Here are the constants: 
https://github.com/influxdata/chronograf/blob/870dbc72d1a8b784eaacad5eeea79fc54968b656/kapacitor/operators.go#L13

File: ui/src/kapacitor/components/RuleGraph.js [hunk 2]
```diff
@@ -88,6 +102,17 @@ export const RuleGraph = React.createClass({
           break
         }
 
+        case 'not equal to': {
+          const width =
+            theOnePercent * (dygraph.yAxisRange()[1] - dygraph.yAxisRange()[0])
+          highlightStart = +rule.values.value - width
```
Comment (goller) [inline ui/src/kapacitor/components/RuleGraph.js]: doesn't seem to work when the values are strings.  It's pretty common to have tickscripts use 'not equal to' or 'equal to' for strings.

Comment (goller) [reply ui/src/kapacitor/components/RuleGraph.js]: Here is a specific example of usage: https://github.com/influxdata/chronograf/issues/865

**Pull request merged**